plugins/pci-pch/README.md

```diff
@@ -0,0 +1,8 @@
+PCI PCH
+=======
+
+Introduction
+------------
+
+This plugin checks if the PCH has DCI enabled. The result will be stored in an
```

Comment:
Probably need to better describe what DCI is here.

Reply:
Fixed. FYI, it's the double-ended USB-3 debug cable thing.